Tests/SentryTests/SentrySpanTests.swift

```diff
@@ -85,6 +87,20 @@ class SentrySpanTests: XCTestCase {
         XCTAssertEqual(serializedChild["parent_span_id"] as? String, span.context.spanId.sentrySpanIdString)
     }
     
+    func testFinishWithUnfinishedChild() {
```

Comment:
L: What about describing the expected outcome of the test in its title?
```suggestion
    func testFinishWithUnfinishedSpanDropsSpan() {
```